src/Stratis.Bitcoin.IntegrationTests.Common/TestHelper.cs

```diff
@@ -37,6 +37,10 @@ public static void WaitLoop(Func<bool> act, string failureReason = "Unknown Reas
 
         public static bool AreNodesSynced(CoreNode node1, CoreNode node2, bool ignoreMempool = false)
         {
+            // If the nodes are at genesis they are considered synced.
+            if (node1.FullNode.Chain.Tip.Height == 0 && node2.FullNode.Chain.Tip.Height == 0)
```

Comment:
What if node1=0 and node2=10

then the condition bellow will fail no?
```
 if (node2.FullNode.BlockStore().GetBlockAsync(node1.FullNode.ChainBehaviorState.ConsensusTip.HashBlock).Result == null)
       return true
```

Reply:
Well yes then the nodes arent in sync? Isnt correct?

Reply:
ahhh yes ofcourse awesome